offerzen/global/employers/employers@4.0.js

```diff
@@ -64,7 +64,13 @@ window.$loaded(function (window, document, $, undefined) {
       const formProperties = Object.fromEntries(formData.entries())
       const role_types = getRoleTypes(formData, formProperties)
 
-      if (form.parsley().validate() /* && role_types.length > 0 */) {
+      if (formProperties.subscribe_to_company_newsletter === false) {
+        formProperties.subscribe_to_hiring_insights = false
+      } else {
+        formProperties.subscribe_to_hiring_insights = true
+      }
```

Comment:
```suggestion
      formProperties.subscribe_to_hiring_insights = (formProperties.subscribe_to_company_newsletter === true)
```

Safer to test truth, as that makes false more likely except in this case. Otherwise it might default to true where the value = 'blah', etc.